tests/yast2_ui/yast2_snapper.pm

```diff
@@ -67,12 +67,13 @@ sub clean_and_quit() {
     # C'l'ose  the snapper module
     send_key "alt-l";
     # Wait until xterm is focussed, delete the directory and close xterm
-    wait_idle;
+    wait_idle 3;
```

Comment:
On 22.01.2016 17:11, Oliver Kurz wrote:
> In tests/yast2_ui/yast2_snapper.pm
> <https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/962#discussion_r50554939>:
> 
>> @@ -67,12 +67,13 @@ sub clean_and_quit() {
>>      # C'l'ose  the snapper module
>>      send_key "alt-l";
>>      # Wait until xterm is focussed, delete the directory and close xterm
>> -    wait_idle;
>> +    wait_idle 3;
> 
> wut?
> 
I remember again: that was my attempt to reuse wait_idle as annoted sleep